File: src/main/java/org/example/gogoma/domain/user/dto/FriendResponse.java
package org.example.gogoma.domain.user.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class FriendResponse {

    private int friendId;

    private String name;

    private String profileImage;

    private int totalDistance;
}

File: app/src/main/java/com/example/gogoma/data/model/FriendResponse.kt
package com.example.gogoma.data.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class FriendResponse (
    val rank: Int,
    val friendId : Int,
    val name : String,
    val profileImage : String?,
    val totalDistance: Int
) : Parcelable